apps/roam/src/components/DiscourseFloatingMenu.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import ReactDOM from "react-dom";
+import { Popover, Menu, MenuItem, Button, Position } from "@blueprintjs/core";
+
+export const DiscourseFloatingMenu = () => (
+  <div id="discourse-floating-menu" className="bottom-right dark">
+    <Popover
+      content={
+        <Menu>
+          <MenuItem
+            text="Send feedback"
+            icon="send-message"
+            onClick={window.birdeatsbug?.trigger}
```

Comment:
Clicking the button did not work for me.  Does this need to be `trigger()`?
Regardless, let's make sure we update the type to match.